front_end/src/reducers/bookingReducer.js

```diff
@@ -2,6 +2,7 @@ import {
   SAVE_BOOKING,
   SET_BOOKING,
   CLEAR_BOOKING,
+  CHANGE_RESERVATION,
```

Comment:
There's a bunch of placeholder code that's been commented out. Please remove if no longer needed or leave a 'TODO' comment if it is needed in future stories.

Reply:
Similarly, it was left there from another story by the author. While we most likely might not need it, I left it there in case the author might need to come back to fix any bugs,